### ISSUE TITLE
Simplify Deals and Ideals core rules

### ISSUE THREAD

**Issue by alice-i-cecile** [Leafwing-Studios/fonts-of-power#82]
## What problem does the current design create?

D&I can be too hard to follow, and challenging to learn.

## What solution would you like?

- reduce payoff to 1D scale from 0-4: extra complexity not worth it
- changing loyalty points to N requires payoff equal to absolute value of level you're shifting to?
- collaborate
- rework D&I action balance

## Additional context

Prompted by FoP Lite work.

**Comment by alice-i-cecile** [comment]
- To be used:
  - As a simpler alternative for small negotiations in FoP core
  - Almost exclusively in FoP Lite
- For each scene, the NPC/group starts at one of four levels:
  - Unwilling, but will tell you what, if anything, would be needed for them to agree to the deal
  - Willing, on a condition
  - Willing, as is
  - Willing, and wants to go above and beyond
- NPCs cannot start at the last level, only one of the first three
- The players can increase the NPC's level by making an appropriate skill check.
  - Flat difficulty 20
  - Use whatever check is appropriate for the situation. What's the NPC's holdup? How can you sweeten the deal? Use normal skill check outcomes for this.
  - On a success, increase the NPC's level by 1
  - On a smashing success, increase the NPC's level by 2
  - You may only make this check once for a given deal. But, if you fail, you may roll into a more full D&I scene.
- Gaining/losing loyalty works as in full D&I
  - I would expect many narrativist tables playing FoP Lite to ignore this, and that's fine I guess
  - Loyalty doesn't hook into simplified D&I directly, but should be considered when determining the NPCs starting level

**Comment by alice-i-cecile** [comment]
@plof27 says she has a functioning version of this for FoP-lite.

**Comment by plof27** [comment]
Completed and merged in 2ddd5a64ca7efe2ccdb62b06a35b39eb366ca0a9 (no PR)
